app/views/tutors/_form.html.erb

```diff
@@ -56,7 +56,7 @@
 
   <div class="row">
     <div class="col-md-4">
-      <%= f.input :phone %>
+      <%= f.input :home_phone %>
```

Comment:
The only reason I didn't is because later today I'm going to open a PR to redo the whole form page to look pretty and reflect the intake form.